Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -510,19 +510,32 @@
 #workflows[140.118] = ['',['RunParkingBPH2022B','HLTDR3_2022','RECONANORUN3_reHLT_2022','SKIMPARKINGBPHRUN3_reHLT']]
 
 ### run3 (2023) ###
-workflows[141.001] = ['',['RunMuon2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.002] = ['',['RunZeroBias2023B','HLTDR3_2023','RECONANORUN3_ZB_reHLT_2023','HARVESTRUN3_ZB_2023']]
-workflows[141.003] = ['',['RunBTagMu2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.004] = ['',['RunNoBPTX2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.005] = ['',['RunHcalNZS2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.006] = ['',['RunHLTPhysics2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.007] = ['',['RunCommissioning2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.008] = ['',['RunJetMET2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.009] = ['',['RunCosmics2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.010] = ['',['RunDisplacedJet2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.011] = ['',['RunEGamma2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.012] = ['',['RunTau2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
-workflows[141.013] = ['',['RunMuonEG2023B','HLTDR3_2023','RECONANORUN3_reHLT_2023','HARVESTRUN3_2023']]
+
```

Comment:
need space here?

Reply:
Hi @srimanob 

   IIUC, you are talking additional empty line between 2023B and 2023C WFs. This is not part of the planned changes and was introduced accidentally.